Twig/NovaeZSEOExtension.php

```diff
@@ -249,12 +249,14 @@ public function getGlobals()
         $metas          = $this->configResolver->getParameter( "default_metas", "nova_ezseo" );
         $links          = $this->configResolver->getParameter( "default_links", "nova_ezseo" );
         $gatracker      = $this->configResolver->getParameter( "google_gatracker", "nova_ezseo" );
+        $anonymizeIp    = $this->configResolver->getParameter( "google_anonymizeIp", "nova_ezseo" );
         $novaeZseo      = [
             "fieldtype_metas_identifier" => $identifier,
             "fieldtype_metas"            => $fieldtypeMetas,
             "default_metas"              => $metas,
             "default_links"              => $links,
-            "google_gatracker"           => $gatracker != "~" ? $gatracker : null
+            "google_gatracker"           => $gatracker != "~" ? $gatracker : null,
+            "google_anonymizeIp"         => $anonymizeIp != "~" ? true : null
```

Comment:
Maybe:
1. let it be true by default (and mentioned that in doc)
2. change this to something like `"google_anonymizeIp"         => $anonymizeIp !== "~" ? !!$anonymizeIp : true`

As for usage of `~` here it feels like this is used the wrong way, but 1. not my area of expertise 2. prior convention all over the place here already.

Reply:
1. I can change it to be true by default, as removing the anonymizeIp should be a opt-out from google's side.
2. I'm not quite sure what "~" does anywho, so I just copied what was already in place.
I can try to update the pull request to fix it.

Reply:
I will indeed do `"google_anonymizeIp" => $anonymizeIp !== "~" ? !!$anonymizeIp : true`